earn/src/components/info/MarketCard.tsx

```diff
@@ -82,6 +94,10 @@ export default function MarketCard(props: MarketCardProps) {
 
   const etherscanLink = `${getEtherscanUrlForChain(activeChain)}/address/${poolAddress}`;
 
+  const lastUpdated = lastUpdatedAt
+    ? formatDistanceToNow(new Date(lastUpdatedAt * 1000), { includeSeconds: false, addSuffix: true })
+    : 'Never';
```

Comment:
Naming?

Reply:
What naming?

Reply:
`lastUpdated` vs `lastUpdatedAt` is confusing.